src/components/countrylist.js

```diff
@@ -0,0 +1,25 @@
+const Country = (props) => {
+  const country = props;
```

Comment:
- [ ] As specified, kindly implement a **redux filter/ search** functionality so as to enable filtering of countries. The filter/ search field should be displayed on the Homepage.